.github/workflows/on-ubuntu-latest.yml

```diff
@@ -0,0 +1,26 @@
+name: on-ubuntu-latest
+
+on:
+  push:
+  schedule:
+    - cron: '0 0 * * 1'  # run every Monday
+
+jobs:
+  on-ubuntu-latest:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+    - name: Initialisation
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: sudo apt-get install -y tk-dev tcl-dev
+      
+    - name: Build OOMMF
+      run: make build-with-all
+
+    - name: Test OOMMF
+      run: make test-all-on-ubuntu-latest
```

Comment:
suggest to rename `test-all-on-ubuntu-latest` to `test-all`

Please add new line at end of line as well.

Reply:
This depends on the first question. Newline at the EOF added.